content/blog/2018/2018-09-02-cccc-rounds-3-4-and-5-leela-with/index.md

```diff
@@ -7,12 +7,13 @@ tags = []
 draft = true
 +++
 
-[ ![](../../images/2018-09-02-cccc-rounds-3-4-and-5-leela-with-
-gjgj98789798799999.png)](https://www.chess.com/computer-chess-championship)
+[![]({{< file "gjgj98789798799999.png" >}})](https://www.chess.com/computer-chess-championship)
```

Comment:
Note that in that case the image will appear in summaries.
I don't know what the image is, but if it's large probably it doesn't look good.

If you just put an image `cover.png` into the directory of the post, then it will appear as a cover image, but not in summaries.

I understand it's a link here, so not sure what is the best solution. Either just remove the link and a cover image, or do that and convert the link to a text link, or leave it as it is if the image is not very large.